department-of-veterans-affairs/va.gov-cms
# VACMS-2513: Adding null check before firing method to prevent ajax failure

## Description
See #2513 
This is being caused by an ajax error that is occurring b/c a method call is being made on an empty variable (`$parent`), here:
```
function _vagovadmin_get_node_parent(Paragraph $paragraph) {
  $parent_node = NULL;
  /** @var Drupal\paragraphs\Entity\Paragraph $parent */
  $parent = $paragraph->getParentEntity();
  if ($parent !== NULL && $parent->getEntityType()->id() === 'paragraph') {
    $parent_node = _vagovadmin_get_node_parent($parent);
  }
  elseif ($parent !== NULL && $parent->getEntityType()->id() === 'node') {
    $parent_node = $parent;
  }
  return $parent_node;
}
```
## Testing done
Steps defined in defect task

## QA steps
- As an admin, go to `node/6785/edit` 
- In main content, add a Q&A section.
- Add some text to the answer portion of the wysiwyg, and collapse it.
- Confirm collapse occurs, then expand, then recollapse.
- Verify that alert modal does not appear, and each interaction occurs as expected.
